octoprint_mrbeam/iobeam/airfilter.py

```diff
@@ -0,0 +1,81 @@
+# singleton
+from octoprint_mrbeam.mrb_logger import mrb_logger
```

Comment:
Please add docstrings to the new methods

Reply:
Ohh yes I missed that, thanks for the hint.